README.md

```diff
@@ -51,52 +61,52 @@ You will need to clone this repository to the machine you want to use to set up
 
 1. First authenticate to Google Cloud:
 
-  ```bash
-  gcloud auth application-default login
-  ```
+    ```bash
+    gcloud auth application-default login
+    ```
 
 2. Create a new project (skip this if you already have a project created):
 
-  ```bash
-  gcloud projects create <your-project-id>
-  ```
+    ```bash
+    gcloud projects create <your-project-id>
+    ```
 
 3. Set the new project as your context for the `gcloud` CLI:
 
-  ```bash
-  gcloud config set project <your-project-id>
-  ```
+    ```bash
+    gcloud config set project <your-project-id>
+    ```
 
 4. Check if your authentication is ok and your project id is set:
 
-  ```bash
-  gcloud projects describe <your-project-id>
-  ```
+    ```bash
+    gcloud projects describe <your-project-id>
+    ```
 
-> __Note:__ You should see your `projectId` listed with an `ACTIVE` state.
+    > __Note:__ You should see your `projectId` listed with an `ACTIVE` state.
 
 5. Setup your unique `.env` variables to be used throughout the setup
 process
 
-  ```bash
-  bash ./scripts/setup-env.sh
-  ```
-
-6. Finally, enable all the needed Google Cloud APIs by running this script:
+    ```bash
+    bash ./scripts/setup-env.sh
+    ```
 
-  ```bash
-  bash ./scripts/enable-api.sh
-  ```
+    >__NOTE:__ During this step you will be prompted for a couple inputs relative to your unique
+    >project. Most inputs will contain defaults that might already be set, in which case go ahead and
+    >press [ENTER] to accept and continue.
+    >
+    >1. The GitHub username/o>rganization. This is the value used above when you cloned your fork.
+    >2. The name of the GitHu>b repository, by default this is set to `gke-github-deployment`.
+    >3. Your unique Google Cl>oud project ID.
+    >4. Defaut region location for Google Cloud setup.
+    >5. A short (3-5 char) identifier for your cloud resources (e.g. gcp).
```

Comment:
Check this new block... you have some spurious '>' characters.

Reply:
will review this, I saw you were able to better alight code blocks within numbered bullets, so was trying to do that

Reply:
Oh I see what happened... fixing